ui/raidboss/data/03-hw/trial/sephirot-ex.ts

```diff
@@ -1,13 +1,322 @@
+    },
+    {
+      // 3ED is Force Against Might orange, 3EE is Force Against Magic, green.
+      id: 'SephirotEx Force Against Gain',
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: ['3ED', '3EE'] }),
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, matches, output) => output.text!({ force: matches.effect }),
+      run: (data, matches) => data.force = matches.effectId,
+      outputStrings: {
+        text: {
+          en: '${force} on you',
+        },
+      },
+    },
+    {
+      // Orange left, Green right. Match color to Force debuff.
+      id: 'SephirotEx Gevurah Chesed',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '1578', capture: false }),
+      alertText: (data, _matches, output) => {
+        // Here and for Chesed Gevurah, if the player doesn't have a color debuff,
+        // they just take moderate AoE damage.
+        // Unlike Flood of Naught (colors) in O4s,
+        // standing center is safe if the user has no debuff.
+        if (data.force)
+          return data.force === '3ED' ? output.left!() : output.right!();
+        return output.aoe!();
+      },
+      outputStrings: {
+        left: Outputs.left,
+        right: Outputs.right,
+        aoe: Outputs.aoe,
+      },
+    },
+    {
+      // Green left, Orange right. Match color to Force debuff.
+      id: 'SephirotEx Chesed Gevurah',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '1579', capture: false }),
+      alertText: (data, _matches, output) => {
+        if (data.force)
+          return data.force === '3EE' ? output.left!() : output.right!();
+        return output.aoe!();
+      },
+      outputStrings: {
+        left: Outputs.left,
+        right: Outputs.right,
+        aoe: Outputs.aoe,
+      },
+    },
+    {
+      id: 'SephirotEx Fiendish Wail',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: '1575', source: 'Sephirot', capture: false }),
+      alertText: (data, _matches, output) => {
+        if (data.force === '3ED' || (!data.force && data.role === 'tank'))
+          return output.getTower!();
+        return output.avoidTower!();
+      },
+      outputStrings: {
+        getTower: {
+          en: 'Get a tower',
+        },
+        avoidTower: {
+          en: 'Avoid towers',
+        },
+      },
+    },
+    {
+      id: 'SephirotEx Da\'at Tethers',
+      type: 'Tether',
+      netRegex: NetRegexes.tether({ id: '0030', capture: false }),
+      suppressSeconds: 30, // The tethers jump around a lot
+      alertText: (data, _matches, output) => {
+        if (data.force === '3EE')
+          return output.magic!();
+        return output.might!();
+      },
+      outputStrings: {
+        might: {
+          en: 'Pass tether; avoid puddles + greens',
+        },
+        magic: {
+          en: 'Grab a tether; get front',
+        },
+      },
```

Comment:
Suggesting a direction as the first thing to say here.

```suggestion
      outputStrings: {
        might: {
          en: 'Get Away, Avoid Puddles + Tethers',
        },
        magic: {
          en: 'Go Front; Get Tether',
        },
      },
```